cognite/client/data_classes/vision.py

```diff
@@ -0,0 +1,255 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Type, Union, cast, get_type_hints
+
+from typing_extensions import get_args
+
+from cognite.client.data_classes import Annotation, ContextualizationJob, ContextualizationJobType
+from cognite.client.data_classes._base import CogniteResource
+from cognite.client.data_classes.annotation_types.images import AssetLink, ObjectDetection, TextRegion
+from cognite.client.data_classes.annotation_types.primitives import VisionResource
+from cognite.client.data_classes.annotations import AnnotationList
+from cognite.client.data_classes.contextualization import JobStatus
+from cognite.client.exceptions import CogniteException
+from cognite.client.utils._case_conversion import resource_to_snake_case
+
+FeatureClass = Union[Type[TextRegion], Type[AssetLink], Type[ObjectDetection]]
+ExternalId = str
+InternalId = int
+
+
+if TYPE_CHECKING:
+
+    from cognite.client import CogniteClient
+
+
+class Feature(str, Enum):
+    TEXT_DETECTION = "TextDetection"
+    ASSET_TAG_DETECTION = "AssetTagDetection"
+    INDUSTRIAL_OBJECT_DETECTION = "IndustrialObjectDetection"
+    PEOPLE_DETECTION = "PeopleDetection"
+    PERSONAL_PROTECTIVE_EQUIPMENT_DETECTION = "PersonalProtectiveEquipmentDetection"
```

Comment:
Something like this?
```suggestion
    PEOPLE_DETECTION = "PeopleDetection"
    # The features below are in beta
    INDUSTRIAL_OBJECT_DETECTION = "IndustrialObjectDetection"
    PERSONAL_PROTECTIVE_EQUIPMENT_DETECTION = "PersonalProtectiveEquipmentDetection"
```

Reply:
[16c43c3](https://github.com/cognitedata/cognite-sdk-python/pull/1014/commits/16c43c35f3557da9e54d0953dea1229764fd02d6)